tests/fixed/concatenation_spacing.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$string = 'Leroy Merlin'.'bar';
+
+$foo = 'Leroy Merlin';
+$bar    = 'bar';
```

Comment:
?

Reply:
That means we don't have a rule for that :)

Reply:
@carusogabriel why are we testing this so?

Reply:
@gmsantos Because we **do** need tests to ensure:

```php
$foo = 'bar';
// over
$foo =    'bar';
```

but, again, isn't in the scope of this PR.

Reply:
I get it, but this test should be included together with the rule, not before.